### ISSUE TITLE
Memory Leak resulting in excessive VRAM usage - Brigand: Oaxaca

### ISSUE THREAD

**Issue by anchorlightforge** [NVIDIAGameWorks/rtx-remix#176]
## Bug Description:
- After attempting to begin a new game in Brigand: Oaxaca, the VRAM usage detected by Remix will spike from about 3.34 GB of VRAM to nearly 14 or 15 GB on the loading screen.
- Tested with DXVK e010672-87 and Bridge 4d6de56-6.
- GPU: Nvidia Geforce RTX 3080
- Driver: Nvidia Studio Driver 531.41
- Game version: Steam - Downgraded to v8.5 due to a CTD on startup in the current build [use Betas tab in game properties]

## Logs: 
- NvRemixBridge logs: 
    - [NvRemixBridge_01.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/11583660/NvRemixBridge_01.log)
     - [NvRemixBridge.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/11583657/NvRemixBridge.log)
- d3d9 logs: 
    - [d3d9_01.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/11583658/d3d9_01.log)
    - [d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/11583659/d3d9.log)

Note: No specific log for Brigand was written, instead using _01 in the naming scheme.  This behavior has been observed with multiple versions of Remix.

## Bug Reproduction Steps:

1. Open Brigand: Oaxaca v8.5 via Steam after installing this version from the Betas tab. (the logs attached occurred with Brigand Windowed.exe)
2. Disable Reflex to bring the menus to a playable framerate.
3. Mark the UI textures to make the menu accessible.
4. Start a new game to create a new player character and attempt to progress to the game world.
5. The Remix tab will, on most attempts, spike from the expected amount of VRAM usage to 10 or more GB of VRAM in a few seconds.
6. On next startup, Brigand will run at 1 FPS and have no UI visible as despite saving from IMGUI, no rtx.conf was written.

**Expected behavior**
1. Game should safely allow the player to create a new character and load into the game world.
2. After clicking through menus and completing player creation, the player will see a prologue text introduction and load into the game world.  On the same hardware, VRAM usage is minimal due to the game's age and no leak was observed.
3. By saving settings changes, UI textures should appear on subsequent runs while Reflex should remain disabled.

### Screenshots
![brigand_spike](https://github.com/NVIDIAGameWorks/rtx-remix/assets/55258193/4e6c8aeb-e16e-413e-acd9-57277e70b67e)

**Comment by NV-LL** [comment]
Hello! Will you please test this on the latest release and let us know if the issue persists? Thank you!

**Comment by anchorlightforge** [comment]
I can confirm that the game's latest version not only boots up now, but does not exhibit this issue.  I will not close this issue until I test game version v8.5 again and make sure that the memory bug is fixed there as well.  Unfortunately I'm hitting some new performance issues that make testing this difficult, which I will report separately as soon as I can.

**Comment by anchorlightforge** [comment]
I'll tentatively say this issue has been fixed-- after some restarts I was able to get the game to run again on v8.5 and while earlier builds of Remix would suffer a massive memory leak when the loading bar for a new game appeared, 0.5.0 appears to have fixed this issue and I am able to begin a new game, though nothing renders.

I'll document what I can for future issues and call this one fixed.  Thanks for wrapping this one up!
